ietf/templates/person/profile.html

```diff
@@ -165,6 +165,21 @@ <h2 class="mt-5">
             <div id="chart-{{ forloop.counter }}">
             </div>
         {% endif %}
+        <h2 class="mt-5" id="reviews-{{ forloop.counter }}">
+            Directorate Reviews In The Last Year <small class="text-muted">({{ person.completed_reviews|length }})</small>
```

Comment:
Do you really want this to show for every user of the datatracker?
I think it would be better show this only if there are reviews to report on.
People who have accounts that are not the kind of person that does reviews (such as policy fellows who have an account so they can register for meetings) will find this line very confusing.

Reply:
Yet another good point of yours. As you may have noticed, I am working  at low intensity on this issue to take into account your previous comments (including re-using the code of ietfauth/views.py about review_overview()

Reply:
maybe mark this draft until you're ready for full review again?